### PR TITLE
Fixing multiple `oneOf`s

### PR DESCRIPTION
**Pull request by sergey-shandar**
Issue #248

### CODE REVIEW

**Comment by vladbarosan** [comment]
also with this , fixes https://github.com/Azure/oav/issues/234 .

File: lib/util/utils.ts [hunk 1]
```diff
@@ -698,8 +699,9 @@ export function allowNullType(entity: Model, isPropRequired?: boolean|{}): Model
     && entity.$ref
     && shouldAcceptNullValue(entity["x-nullable"], isPropRequired)) {
     const savedEntity = entity
-    entity = {}
-    entity.oneOf = [savedEntity, { type: "null" }]
+    entity = {
+      anyOf: [savedEntity, { type: "null" }]
```
Comment (vladbarosan) [inline lib/util/utils.ts]: >anyOf [](start = 6, length = 5)

is this the only place where we used to make a oneOf between the type and null ?

Comment (sergey-shandar) [reply lib/util/utils.ts]: The one that I'm aware of that cause problems. It looks like other cases where we use `oneOf` should be good.

Comment (vladbarosan) [reply lib/util/utils.ts]: also I was wondering, are we ready to enable tslint on the project 

---
In reply to: [196611051](https://github.com/Azure/oav/pull/263#discussion_r196611051) [](ancestors = 196611051)

Comment (sergey-shandar) [reply lib/util/utils.ts]: @vladbarosan tslint is enabled already

**Pull request merged**